### PR TITLE
fix/dashboard-input

### PR DESCRIPTION
**Pull request by KazemHe**
# Description
<Please describe the suggested changes>
change the input design in the dashboardpage   issue #167

## screenshots

image

### CODE REVIEW

**Comment by shootermv** [comment]
Can you please remove packge-lock file?

**Comment by KazemHe** [comment]
> Can you please remove packge-lock file?

Done

File: yarn.lock
Comment (NoamGaash) [inline yarn.lock:1]: not sure why yarn.lock changed

**Review by NoamGaash — APPROVED** [review]
lgtm

**Review by NoamGaash — APPROVED** [review]

**Review by shootermv — APPROVED** [review]

**Comment by NoamGaash** [comment]
run `git checkout main yarn.lock` to reset the `yarm.lock` file to main.

**Comment by shootermv** [comment]
Thanks for your contribution @KazemHe!

**Comment by KazemHe** [comment]
> Thanks for your contribution @KazemHe!

My pleasure ! hope to keep contributing to the project .

**Pull request merged**